src/test/java/org/apache/commons/text/lookup/DateStringLookupTest.java

```diff
@@ -42,9 +43,16 @@ void testBadFormat() {
 
     @Test
     void testDefault() throws ParseException {
-        final String formatted = DateStringLookup.INSTANCE.apply(null);
-        DateFormat.getInstance().parse(formatted); // throws ParseException
+        Locale prev = Locale.getDefault();
+        try {
+            // ensure that date-parser uses English/American presets
```

Comment:
Why is this important for this test?

That's not how you set with Locale's anyway, this component depends on JUnit Pioneer for this type of stuff (see `@DefaultLocale`)